__tests__/Auth0Client/getTokenSilently.test.ts

```diff
@@ -179,13 +180,24 @@ describe('Auth0Client', () => {
 
       await getTokenSilently(auth0);
 
-      assertPost('https://auth0_domain/oauth/token', {
-        redirect_uri: TEST_REDIRECT_URI,
-        client_id: TEST_CLIENT_ID,
-        code_verifier: TEST_CODE_VERIFIER,
-        grant_type: 'authorization_code',
-        code: TEST_CODE
-      });
+      assertPost(
+        'https://auth0_domain/oauth/token',
+        {
+          redirect_uri: TEST_REDIRECT_URI,
+          client_id: TEST_CLIENT_ID,
+          code_verifier: TEST_CODE_VERIFIER,
+          grant_type: 'authorization_code',
+          code: TEST_CODE
+        },
+        {
+          'Auth0-Client': btoa(
+            JSON.stringify({
+              name: 'auth0-spa-js',
+              version: version
+            })
+          )
```

Comment:
As we're adding coverage here, it would be worth adding the same assertions on the headers for the test that follows ("..when using refresh tokens"). Looks like an omission testing for one but not the other, even though in reality we know both are covered.

Reply:
Ye I thought about it, wasn't sure and decided to not pollute all tests with it. But I think I can add it.